packages/@react-spectrum/card/src/BaseLayout.tsx

```diff
@@ -14,9 +14,17 @@ import {Direction, KeyboardDelegate, Node} from '@react-types/shared';
 import {GridCollection} from '@react-stately/grid';
 import {InvalidationContext, Layout, LayoutInfo, Rect, Size} from '@react-stately/virtualizer';
 import {Key} from 'react';
+import {Scale} from '@react-types/provider';
 
 export interface BaseLayoutOptions {
-  collator?: Intl.Collator
+  collator?: Intl.Collator,
+  // TODO: is this valid or is scale a spectrum specific thing that should be left out of the layouts?
+  scale?: Scale,
```

Comment:
This really is a question of whether or not we would expect users to create their own layouts that extend off this layout. Since this lives in the `@react-spectrum` package I'm leaning towards `scale` being an acceptable option to expect someone to provide in their layout